### PR TITLE
feat(SuiteTalk): Upgraded to SuiteTalk 2024.1

### PR DESCRIPTION
**Pull request by srikanth-cloudextend**

### CODE REVIEW

**Review by jag-cloudextend — APPROVED** [review]

File: Celigo.ServiceManager.NetSuite.REST/src/Celigo.ServiceManager.NetSuite.REST.csproj [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
     <PropertyGroup>
-      <TargetFramework>net6.0</TargetFramework>
-      <Version>2.0.0</Version>
+      <TargetFramework>net8.0</TargetFramework>
```
Comment (sameera) [inline Celigo.ServiceManager.NetSuite.REST/src/Celigo.ServiceManager.NetSuite.REST.csproj:4]: Was the upgrade mandatory? In the past we've encountered customers who needed the new WSDL but didn't have the option to upgrade their .net code base. So, I'd be cautious about bumping up the version.

**Review by sameera — APPROVED** [review]
Approved but with a comment regarding version upgrade. Merge at the team's discresion.

File: Celigo.ServiceManager.NetSuite.REST/src/Celigo.ServiceManager.NetSuite.REST.csproj (hunk 1 above)
Comment (srikanth-cloudextend) [reply Celigo.ServiceManager.NetSuite.REST/src/Celigo.ServiceManager.NetSuite.REST.csproj]: We had an internal discussion and we are moving this back to the earlier .net versions

File: Samples/ThreeStepAuthMVC/ThreeStepAuthMVC.csproj [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk.Web">
 
     <PropertyGroup>
-        <TargetFramework>net6.0</TargetFramework>
+        <TargetFramework>net8.0</TargetFramework>
```
Comment (pankaj-cloudextend) [inline Samples/ThreeStepAuthMVC/ThreeStepAuthMVC.csproj:4]: Let's revert this as well

**Review by jag-cloudextend — APPROVED** [review]

**Review by pankaj-cloudextend — APPROVED** [review]

**Review by vka-cloudextend — APPROVED** [review]
Reviewed the pipeline file

**Pull request merged**